### PR TITLE
Add new interfaces for publication links

### PR DESCRIPTION
**Pull request by kevin-jueni**

### CODE REVIEW

File: postman/collections/denner-api.postman-v2.json [hunk 1]
```diff
@@ -632,7 +632,7 @@
...
 						},
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: What changed here?

File: postman/collections/denner-api.postman-v2.json [hunk 2]
```diff
@@ -5665,7 +5813,7 @@
 					],
 					"request": {
 						"url": {
-							"raw": "https://requestb.in/{your-request.bin-id}?inspect=",
+							"raw": "https://requestb.in/{your-reqwstb.in-id}?inspect=",
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: I don't think this changes is intended.

File: postman/collections/denner-api.postman-v2.json [hunk 3]
```diff
@@ -1537,36 +1537,6 @@
...
-							}
-						],
-						"body": {
-							"mode": "raw",
-							"raw": ""
-						},
-						"description": "Delete an article (Artikel) from a print publication (Print-Werbemittel)"
-					},
-					"response": []
-				},
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: Why did you delete this operation?

File: postman/collections/denner-api.postman-v2.json [hunk 4]
```diff
@@ -2190,6 +2160,184 @@
...
+									"    id = (jsonData) ? jsonData.id : null;",
+									"    ",
+									"postman.setEnvironmentVariable(\"publication_article_id\", id);",
+									"",
+									"tests[\"Setup new publication_article_id in environment\"] = !!id;",
+									""
+								]
+							}
+						}
+					],
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: This script does not work for links (if anything should be saved to an environment var, it's a link).

File: postman/collections/denner-api.postman-v2.json [hunk 5]
```diff
@@ -2190,6 +2160,184 @@
...
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"id\": \"{{link_id}}\",\n\t\"name\": \"Hans\",\n\t\"valid_to\": \"2017-08-28\"\n}"
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: We should not allow to provide the ID.

File: postman/collections/denner-api.postman-v2.json [hunk 6]
```diff
@@ -2190,6 +2160,184 @@
...
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"name\": \"Hans 2\",\n\t\"valid_to\": \"2017-08-29\"\n}"
+						},
+						"description": "Update Links's details from a print publication by listing (Print-Werbemittel)"
```
Comment (ivan-wolf) [inline postman/collections/denner-api.postman-v2.json]: Write "Link's" lowercase.

For the other operation you don't mention that links only work for publications "by listing". I think we don't need to mention it here.

**Review by ivan-wolf — CHANGES_REQUESTED** [review]

File: postman/collections/denner-api.postman-v2.json (hunk 1 above)
Comment (curzio-della-santa) [reply postman/collections/denner-api.postman-v2.json]: The year from 2015 to 2017, I'll rollback

**Pull request merged**